importer/src/main.rs

```diff
@@ -63,7 +65,13 @@ fn main() {
         }
 
         if job.raw_to_map {
-            utils::raw_to_map(&name, job.use_fixes);
+            let thread_name = name.clone();
+            //Fetch use_fixes after already loading into job in order to avoid move compile error
+            let use_fixes = !args.enabled("--nofixes");
```

Comment:
args isn't mean to be used after `.done()`. You can just do `let use_fixes = job.use_fixes` here.